README.md

```diff
@@ -1,29 +1,37 @@
 # imdb_dashboard
 
+The IMDb dashboard is primarily targeted towards movie producers to present a consolidated crisp view of the average ratings and runtime for movies by genres and regions with interactive abilities to help them choose and plan their next movie.
+
 ## Dashboard description
 
-Our dashboard consists of one web page that shows 4 main reactive plots:
+Our dashboard consists of one web page that shows overall summary and 4 main reactive plots:
+
+- The high level totals
 
-- The top rated movies by genre
+These are a set of 4 numeric totals showing the total count of movies, the average runtime across all movies, the total number of actors across all movies, and the average rating of all movies based on the filters applied.
```

Comment:
- if the first chart is "the proportion of movies by genre" we should change it back to bar chart

Reply:
As discussed, updated to "The distribution of movies by genre"